class InternalException(Exception):
    def __self__(msg):
        super.__self__(msg)

